src/TurbulenceClosures/turbulence_closure_implementations/scale_invariant_smagorinsky.jl

```diff
@@ -22,11 +22,18 @@ struct ScaleInvariantSmagorinsky{TD, AP, FT, P, UF} <: AbstractScalarDiffusivity
     function ScaleInvariantSmagorinsky{TD, AP, FT}(averaging, Pr; update_frequency = 5) where {TD, AP, FT}
         Pr = convert_diffusivity(FT, Pr; discrete_form=false)
         P = typeof(Pr)
-        update_frequency = Integer(update_frequency)
-        return new{TD, AP, FT, P, Integer}(averaging, Pr, update_frequency)
+        update_frequency = Int(update_frequency)
```

Comment:
this is probably an interval (I'm guessing, in iterations) if its an integer. The frequency is 1 / interval